src/services/alloc/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include_directories(${GOTCHA_INCLUDE_DIR})
+
+set(CALIPER_ALLOC_SOURCES
+        ../../caliper/AllocTracker.cpp
```

Comment:
Why are we compiling this here and not in src/caliper?

Reply:
Sometimes using CLion has its quirks...